### PR TITLE
OHRM-288 phpunit: fix unit tests issue for php7.3

### PR DESCRIPTION
**Pull request by samanthajayasinghe**

### CODE REVIEW

File: symfony/plugins/orangehrmAttendancePlugin/lib/dao/AttendanceDao.php [hunk 1]
```diff
@@ -233,7 +233,7 @@ public function deleteAttendanceRecords($attendanceRecordId) {
 
             $result = $q->execute();
 
-            if (count($result) > 0) {
+            if (empty($result)) {
```
Comment (Damith88) [inline symfony/plugins/orangehrmAttendancePlugin/lib/dao/AttendanceDao.php]: shouldn't this be !empty() instead?

File: symfony/plugins/orangehrmCorePlugin/test/components/LabelCellTest.php [hunk 2]
```diff
@@ -32,7 +32,7 @@ protected function tearDown() {
 
     public function test__toString() {
         $dataObject = new LabelCellTestDataObject();
-        $dataObject->getName = create_function('', "return 'Kayla Abbey';");
+        $dataObject->getName = 'Kayla Abbey';
```
Comment (Damith88) [inline symfony/plugins/orangehrmCorePlugin/test/components/LabelCellTest.php]: shouldn't  this be `$dataObject->getName = function() {
	return 'Kayla Abbey';
};`

File: symfony/plugins/orangehrmAttendancePlugin/lib/dao/AttendanceDao.php (hunk 1 above)
Comment (samanthajayasinghe) [reply symfony/plugins/orangehrmAttendancePlugin/lib/dao/AttendanceDao.php]: yeh really .. my bad .. :(

Comment (samanthajayasinghe) [reply symfony/plugins/orangehrmAttendancePlugin/lib/dao/AttendanceDao.php]: sure ..

File: symfony/plugins/orangehrmCorePlugin/test/components/LabelCellTest.php (hunk 2 above)
Comment (samanthajayasinghe) [reply symfony/plugins/orangehrmCorePlugin/test/components/LabelCellTest.php]: I thought the same .. but actually not sure why they create that function inside the tests

File: symfony/plugins/orangehrmAttendancePlugin/lib/dao/AttendanceDao.php (hunk 1 above)
Comment (samanthajayasinghe) [reply symfony/plugins/orangehrmAttendancePlugin/lib/dao/AttendanceDao.php]: done

File: symfony/plugins/orangehrmCorePlugin/test/components/LabelCellTest.php (hunk 2 above)
Comment (Damith88) [reply symfony/plugins/orangehrmCorePlugin/test/components/LabelCellTest.php]: yeah, it seems function is not used, even though defined.

**Pull request merged**